src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp

```diff
@@ -167,11 +167,17 @@ bool ControllerScriptEngineLegacy::initialize() {
 }
 
 void ControllerScriptEngineLegacy::shutdown() {
-    callFunctionOnObjects(m_scriptFunctionPrefixes, "shutdown");
+    // There is no js engine if the mapping was not loaded from a file but by
+    // creating a new, empty mapping LegacyMidiControllerMapping with the wizard
+    if (m_pJSEngine) {
+        callFunctionOnObjects(m_scriptFunctionPrefixes, "shutdown");
+    }
```

Comment:
otherwise we'd hit debug assertions in [`callFunctionOnObjects`](https://github.com/mixxxdj/mixxx/blob/4810a5c6b64128ca1f8f84e0d482ddacf0429ff4/src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp#L24-L31) and [`ControllerScriptEngineBase::shutdown()`](https://github.com/mixxxdj/mixxx/blob/4810a5c6b64128ca1f8f84e0d482ddacf0429ff4/src/controllers/scripting/controllerscriptenginebase.cpp#L55-L58) when the new mapping applied.

Reply:
(peviously/2.3 both methods simly returned when there was no js engine)